src/core/reference.js

```diff
@@ -20,6 +28,14 @@
  */
 
 /**
+ * Creates and names a new constant variable. A variable is a container for a value.
+ * Variables that are declared with <a href="#/p5/const">const</a> will have block-scope.
+ * This means that the variable only exists within the <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/block">
+ * block</a> that it is created within.
+ * <a href="#/p5/let">const</a> is different from <a href="#/p5/let">let</a> in that
+ * variables declared with <a href="#/p5/let">const</a> cannot be reassigned or redeclared.
```

Comment:
Are these links here all supposed to link to `let`?

Reply:
Haha thanks!